### ISSUE TITLE
click_record file is empty

### ISSUE THREAD

**Issue by tashrifbillah** [AMP-SCZ/eeg-qc-dash#6]
The above will prevent firing of initial callback. It was observed only once at `rc-predict-lochness`. If it happens again, insert:

```python
            # for the unlikely case where click_record is empty
            if not old_click:
                old_click=-1
```

after https://github.com/AMP-SCZ/eeg-qc-dash/blob/b942d4a98925e6bb35fbdcfb6409349522079c8a/app.py#L202

**Comment by tashrifbillah** [comment]
No longer relevant!
